packages/ref-doc/src/extractor.ts

```diff
@@ -82,6 +87,11 @@ export function extractRefDocs(program: Program, filterToNamespace: string[] = [
             namespaceDoc.unions.push(extractUnionRefDocs(program, union as any));
           }
         },
+        scalar(scalar) {
+          if (scalar.name !== undefined) {
```

Comment:
can this happen?

Reply:
Good point. I followed the existing pattern but yeah the TypeScript shows it's not optional.